cves/CVE-2016-3087.yml

```diff
@@ -207,4 +228,6 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those in the software
     engineering industry would find interesting.
-  answer:
+  answer: |
+    A mistake was made during implementation. The developer simply did not validate
```

Comment:
Where could this have been headed off? This could be punched up a bit to be more interesting.